eos-upgrade-eos2-to-eos3

```diff
@@ -106,16 +106,17 @@ if ! $FORCE; then
     fi
 fi
 
-# Stop any updates in progress
-killall ostree-daemon || true
+# Stop any updates in progress, and prevent the timer from restarting them
+systemctl stop eos-updater.timer || true
+systemctl stop eos-updater.service || true
```

Comment:
Oh, I had not noticed that difference.  But I'm confused why when I was manually testing, I did not see an error when I tried `systemctl stop eosupdater.timer` / `systemctl stop eos-updater.service`.  Would there be any difference in my case where I started with 2.3.4 and then ostree upgraded to 2.6.10?